src/schema/v2/me/__tests__/collectArtistsMutation.test.ts

```diff
@@ -0,0 +1,84 @@
+import { runAuthenticatedQuery } from "schema/v2/test/utils"
+
+describe("collectArtistsMutation", () => {
+  const mutation = `
+    mutation {
+      collectArtists(input: { artistIDs: ["artist_id_1", "artist_id_2"]}) {
+        userInterests {
+          category
+          interest {
+            ... on Artist {
+              name
+            }
+          }
+        }
+        me {
+          name
+        }
+      }
+    }
+  `
+
+  const userInterest = {
+    category: "collected_before",
+    interest: {
+      birthday: "10.10.2000", // without birthday it resolves to GeneType
```

Comment:
I wish all our API responses included a `type' or something so didn't have to do weird introspection like these